zerver/worker/queue_processors.py

```diff
@@ -539,7 +539,9 @@ def consume(self, event: Mapping[str, Any]) -> None:  # nocoverage
 class FetchLinksEmbedData(QueueProcessingWorker):
     def consume(self, event: Mapping[str, Any]) -> None:
         for url in event['urls']:
+            start_time = time.time()
             url_preview.get_link_embed_data(url)
+            logging.info("Time spent on get_link_embed_data for %s: %s", url, time.time() - start_time)
 
         message = Message.objects.get(id=event['message_id'])
```

Comment:
I guess in theory time could go into the code below, but it's likely not significant.  We can add logging later as needed.